src/utils/include/com/amazonaws/kinesis/video/utils/Include.h

```diff
@@ -69,6 +69,7 @@ extern "C" {
 #define STATUS_EXPONENTIAL_BACKOFF_RETRIES_EXHAUSTED STATUS_UTILS_BASE + 0x0000002b
 #define STATUS_THREADPOOL_MAX_COUNT                  STATUS_UTILS_BASE + 0x0000002c
 #define STATUS_THREADPOOL_INTERNAL_ERROR             STATUS_UTILS_BASE + 0x0000002d
+#define STATUS_UTILS_INVALID_THREAD_PARAMS_VERSION   STATUS_UTILS_BASE + 0x0000002e
```

Comment:
(nit) no need to include "UTILS" in name.

Reply:
Was mulling over that one.. I referenced the "tags" implementation and its status codes all had it. I'll remove it since  there's only 1 instance of this anyway.